packages/financial-templates-lib/package.json

```diff
@@ -22,7 +22,6 @@
   },
   "devDependencies": {
```

Comment:
I think we need to actually add `web3` as a runtime dep of this package even though it currently isn't there.

Reply:
Yeah at the very least its imported in ExpressionPriceFeed